tfx/components/trainer/executor.py

```diff
@@ -278,8 +292,12 @@ def Do(self, input_dict: Dict[Text, List[types.Artifact]],
     tf.estimator.train_and_evaluate(training_spec['estimator'],
                                     training_spec['train_spec'],
                                     training_spec['eval_spec'])
-    absl.logging.info('Training complete.  Model written to %s',
-                      fn_args.serving_model_dir)
+
```

Comment:
I wonder if there is a way to provide both working_dir and model_dir to trainer_spec, thus we don't need to manually do the split in your next pr (model artifact refactor)

Reply:
if i recall correcly, the estimator api only writes a model (and its log information) to one directory, but i'll follow up on this in a bit